doc/platforms/index.rst

```diff
@@ -1,15 +1,17 @@
 .. _platforms-index:
 
-Platform-Specific Notes
-=======================
-
-Contents:
-
-.. toctree::
-   :hidden:
+Platforms-Specific Notes
+========================
 
 .. toctree::
    :maxdepth: 1
-   :glob:
 
-   **
+   cray
+   macosx
+   cygwin
+   knl
+   infiniband
+   udp
+   ibm
+   marenostrum
+   sgi
```

Comment:
Here's where I realize and have to confess that I didn't actually look at the order the first time through, just liked the idea of sorting based on priority.  I might be tempted to reorder to the following, primarily to take cray out of the #1 slot and emphasize the notes that are likely to be useful to a broader group of people:

* macosx (probably most common of the important cases here)
* cray (important to us)
* udp (most common network)
* infiniband (most common network)
* knl
* cygwin
* ...

Or, we could add additional categories which might make the lists a bit easier to visually parse.  For example:

major platforms
----------------
* macosx
* cray
* knl

networks
---------
* udp
* infiniband

older platforms (may be out of date)
-------------------------------------
* ibm
* marenostrum
* sgi